src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheckTest.java

```diff
@@ -45,7 +45,6 @@ protected String getPackageLocation() {
     @Test
     public void testBadOverrideFromObject() throws Exception {
         final DefaultConfiguration checkConfig = createModuleConfig(MissingOverrideCheck.class);
-        checkConfig.addAttribute("javaFiveCompatibility", "false");
```

Comment:
we do not modify Check configuration in test methods.
such lines will stay there forever, and may be removed at the very end when parser and verify methods will be VERY reliable.

new verify method should compare `checkConfig` and what we get from Input file.

Reply:
I did not modify Check configuration. I just removed default value. The property ```javaFiveCompatibility``` is ```false``` by default in ```MissingOverrideCheck```.

Reply:
We are now testing with the InlineConfig, just for this particular test method. @timurt @romani

Reply:
@shashwatj07 , better to not change test methods, sometimes they looks weird and a bit over requirement, but it is ok, sometime weird stuff is required to reach 100% coverage.